ui/warlock/sim.ts

```diff
@@ -18,44 +18,30 @@ const SPEC_CONFIG = registerSpecConfig(Spec.SpecWarlock, {
 	// All stats for which EP should be calculated.
 	epStats: [
 		// Primary
-		Stat.StatHealth,
 		Stat.StatMana,
 		// Attributes
-		Stat.StatStrength,
-		Stat.StatStamina,
 		Stat.StatIntellect,
 		Stat.StatSpirit,
-		Stat.StatAgility,
-		// Physical
-		Stat.StatAttackPower,
-		Stat.StatMeleeCrit,
-		Stat.StatMeleeHit,
 		// Spell
 		Stat.StatSpellPower,
 		Stat.StatSpellDamage,
 		Stat.StatSpellHit,
 		Stat.StatSpellCrit,
-		Stat.StatSpellHaste,
 		Stat.StatFirePower,
 		Stat.StatShadowPower,
 		Stat.StatMP5,
 	],
-	epPseudoStats: [PseudoStat.PseudoStatCastSpeedMultiplier],
```

Comment:
Hm this _should_ work correctly. This is also what we use in SoD to provide the cast speed multiplier (effectively haste that functions differently because haste isn't in classic)

Reply:
Oh but you know what cast speed isn't on vanilla gear so that's actually fine